shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -22,6 +22,20 @@ @interface AppDelegate ()
 
 @implementation AppDelegate
 
+- (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
+{
+  NSURL* URL= [NSURL fileURLWithPath: filePathString];
+  assert([[NSFileManager defaultManager] fileExistsAtPath: [URL path]]);
```

Comment:
Apple sample code is notoriously bad.

You should definitely remove the assert though.